src/mmda/types/annotation.py

```diff
@@ -150,25 +152,30 @@ def type(self, type: Union[str, None]) -> None:
 
 
 class SpanGroup(Annotation):
-
     def __init__(
-            self,
-            spans: List[Span],
-            box_group: Optional[BoxGroup] = None,
-            id: Optional[int] = None,
-            doc: Optional['Document'] = None,
-            metadata: Optional[Metadata] = None,
+        self,
+        spans: List[Span],
+        box_group: Optional[BoxGroup] = None,
+        id: Optional[int] = None,
+        doc: Optional["Document"] = None,
+        metadata: Optional[Metadata] = None,
+        allow_overlap: Optional[bool] = False,
```

Comment:
Same story here as `BoxGroup` -- `False` as the default is a breaking change in behavior, and there's no telling how much data we already have that violates it.